wallets/keplr-extension/src/extension/registry.ts

```diff
@@ -22,6 +22,34 @@ export const keplrExtensionInfo: Wallet = {
     source: 'Request rejected',
   },
   connectEventNamesOnWindow: ['keplr_keystorechange'],
+  supportedChains: [
+    'agoric',
+    'akash',
+    'axelar',
+    'bostrom',
+    'certik',
+    'cosmoshub',
+    'cryptoorgchain',
+    'emoney',
+    'evmos',
+    'gravitybridge',
+    'irisnet',
+    'ixo',
+    'juno',
+    'kava',
+    'osmosis',
+    'persistence',
+    'regen',
+    'secretnetwork',
```

Comment:
add any one of these chains to test if it still works, then we've succeeded — using the `useChains` hook

Reply:
@theothersideofgod 

test

1. checkout the branch of this PR, in the `useChains` example in the package
1. a) add a chain like osmosis that is "supported"
1. b) add a chain like noise that is not supported

both should work, and provide an address at least, with no errors from cosmos-kit